.node-version

```diff
@@ -0,0 +1 @@
+5.7.1
```

Comment:
I would need write access to your fork, no? It's the same content and the filename is `.nvmrc`. https://github.com/creationix/nvm#nvmrc